databricks/koalas/window.py

```
@@ -58,6 +70,70 @@ def __getattr__(self, item: str) -> Any:
+        Series or DataFrame
+            Returned object type is determined by the caller of the expanding
+            calculation.
+
+        See Also
+        --------
+        Series.expanding : Calling object with Series data.
+        DataFrame.expanding : Calling object with DataFrames.
+        DataFrame.count : Count of the full DataFrame.
+
+        Examples
+        --------
+        >>> s = ks.Series([2, 3, float("nan"), 10])
+        >>> s.expanding().count()
+        0    1.0
+        1    2.0
+        2    2.0
+        3    3.0
+        Name: 0, dtype: float64
+
+        >>> s.to_frame().expanding().count()
+             0
+        0  1.0
+        1  2.0
+        2  2.0
+        3  3.0
+        """
+        from databricks.koalas import DataFrame, Series
+
+        if isinstance(self.kdf_or_kser, Series):
+            kser = self.kdf_or_kser
+            # TODO: is this a bug? min_periods is not respected in expanding().count() in pandas.
+            # scol = F.when(
+            #     F.row_number().over(self._window) > self._min_periods,
+            #     F.count(kser._scol).over(self._window)
+            # ).otherwise(F.lit(None))
+            scol = F.count(kser._scol).over(self._window)
+            return kser._with_new_scol(scol).astype('float64').rename(kser.name)
+        elif isinstance(self.kdf_or_kser, DataFrame):
+            # TODO: deduplicate with other APIs in expanding.
+            kdf = self.kdf_or_kser
+            applied = []
+            for column in kdf.columns:
+                applied.append(kdf[column].expanding(self._min_periods).count())
+
+            sdf = kdf._sdf.select(
+                kdf._internal.index_scols + [c._scol for c in applied])
+            internal = kdf._internal.copy(
+                sdf=sdf,
+                data_columns=[c._internal.data_columns[0] for c in applied],
+                column_index=[c._internal.column_index[0] for c in applied])
+            return DataFrame(internal)
```

should raise an error when else case?